com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumUnityAdWrapper.kt

```diff
@@ -0,0 +1,225 @@
+        }
+        bannerLayout?.gravity = bannerGravityPosition
+
+        // Attach the banner layout to the activity.
+        val pixels = displayDensity
+        try {
+            val bannerAd = asBanner()
+            when (if (bannerAd?.getSize() != null) bannerAd.getSize() else HeliumBannerSize.STANDARD) {
+                HeliumBannerSize.LEADERBOARD -> bannerAd?.layoutParams = getBannerLayoutParams(pixels, LEADERBOARD_WIDTH, LEADERBOARD_HEIGHT)
+                HeliumBannerSize.MEDIUM -> bannerAd?.layoutParams = getBannerLayoutParams(pixels, MEDIUM_WIDTH, MEDIUM_HEIGHT)
+                HeliumBannerSize.STANDARD -> bannerAd?.layoutParams = getBannerLayoutParams(pixels, STANDARD_WIDTH, STANDARD_HEIGHT)
+                null -> TODO()
+            }
+
+            // Attach the banner to the banner layout.
+            bannerLayout?.addView(bannerAd)
+            activity.addContentView(
+                bannerLayout,
+                FrameLayout.LayoutParams(
+                    FrameLayout.LayoutParams.MATCH_PARENT,
+                    FrameLayout.LayoutParams.MATCH_PARENT
+                )
+            )
+
+            // This immediately sets the visibility of this banner. If this doesn't happen
+            // here, it is impossible to set the visibility later.
+            bannerAd?.visibility = View.VISIBLE
+
+            // This affects future visibility of the banner layout. Despite it never being
+            // set invisible, not setting this to visible here makes the banner not visible.
+            bannerLayout?.visibility = View.VISIBLE
+        } catch (ex: Exception) {
+            Log.w(TAG, "Helium encountered an error calling banner load() - " + ex.message)
+        }
+    }
+
+    private fun destroyBannerLayout() {
+        if (ad is HeliumBannerAd && bannerLayout != null) {
+            bannerLayout?.removeAllViews()
+            bannerLayout?.visibility = View.GONE
+        }
+    }
+
+    private fun getBannerLayoutParams(
+        pixels: Float,
+        width: Int,
+        height: Int
+    ): FrameLayout.LayoutParams {
+        return FrameLayout.LayoutParams((pixels * width).toInt(), (pixels * height).toInt())
+    }
+
+    private val displayDensity: Float
+        get() {
+            try {
+                return activity.resources.displayMetrics.density
+            } catch (ex: Exception) {
+                Log.w(TAG, "Helium encountered an error calling getDisplayDensity() - ${ex.message}")
+            }
+            return DisplayMetrics.DENSITY_DEFAULT.toFloat()
+        }
+
+    companion object {
+        private const val TAG = "HeliumUnityAdWrapper"
+        private const val LEADERBOARD_WIDTH = 728
+        private const val LEADERBOARD_HEIGHT = 90
+        private const val MEDIUM_WIDTH = 300
+        private const val MEDIUM_HEIGHT = 250
+        private const val STANDARD_WIDTH = 320
+        private const val STANDARD_HEIGHT = 50
```

Comment:
Maybe we could use `Pair` to represent width and height as 2 properties of a thing: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-pair/.